content/_index.md

```diff
@@ -1,33 +1,40 @@
 ---
 title: Welcome
-menu: "main"
+menu:
+    main:
+        Pre: star
 weight: -10
+mainSectionId: banner
 ---
 
 <div class="content">
-    <header>
+    <header class="major">
         <h1>Volkamer Lab</h1>
         <p>In silico toxicology and structural bioinformatics</p>
     </header>
 
-The group of JProf. Volkamer is working on novel methods for in-silico toxicology predictions.
+At Volkamer Lab, we develop methods at the interface between structural bioinformatics and cheminformatics, mostly applied in the context of computer-aided drug design.
 
-> Breaking news: join us in our [collective effort](https://github.com/volkamerlab/covid19-SBapproach) to understand Covid-19 better
+On one hand, we investigate [structure-based methods](/research/structure-based/) for active site assessment, including [binding site comparison](/research/structure-based/binding-sites/), [pharmacophore elucidation](/research/structure-based/pharmacophores/) and off-target prediction; pocket-centric [fragment-based design](/projects/kinfraglib/) as well as [deep learning enhanced virtual screening pipelines](/projects/deeplearning-vs/). Besides target-independent approaches, we focus our developments on [kinases](/research/structure-based/kinases/), one of the major classes of therapeutic targets.
 
+On the other hand, we study ligand-based methods, including [machine learning methods for activity and toxicity prediction](/research/machine-learning/) and apply them in translational projects. Special focus is set on integrating novel techniques taking the [applicability and reliability into account](/projects/knowtox/), and the [interpretability of deep learning methods](/projects/cytotox/). Our major goal here is the establishment of alternative _in silico_ methods to determine the risk of compounds and their harmful effects on humans, animals, plants and environment (see <a href="https://www.bb3r.de/en/index.html" target="_blank" class="external">BB3R platform</a>).
```

Comment:
"major goal here is the establishment of  alternative in silico methods": To me it is not clear for what we want to establish alternative in silico methods. 
It sound like alternatives to already existing in silico methods. However, I assume you rather meant alternatives to in vitro/in vivo testing?

Reply:
@AndreaVolkamer, I need your feedback here :)

Reply:
`major goal here is the establishment of _in silico_ methods`